app/src/main/java/com/example/githubsearcher/utils/Extensions.kt

```diff
@@ -11,7 +11,7 @@ fun View.hideKeyboard() {
     imm.hideSoftInputFromWindow(windowToken, 0)
 }
 
-fun ImageView.showImageWithUrl(url: String){
+fun ImageView.showImageWithUrl(url: String?){
     Glide.with(context)
         .load(url)
         .into(this)
```

Comment:
```suggestion
    url?.let {
        Glide.with(context)
            .load(it)
            .into(this)
    }
```
Null safety için bu şekilde yapman daha doğru olur.